docs/ruleset_functions/get_proposed_hvac_modeled_with_virtual_cooling.md

```diff
@@ -0,0 +1,40 @@
+        # The line below will check if a virtual HVAC system was modeled in the proposed. If the hvac_p is not in the U_RMR then its a virtual system.
+        - Check if the hvac system in the proposed design model is NOT in the U_RMR: `if hvac_p.id not in hvac_list_u:`
+            - Reset virtual_cooling_modeled_check_p boolean variable: `virtual_cooling_modeled_check_p = FALSE`
+            # Below checks if a cooling system specifically was modeled (could just be heating for this hvac system) for the vitual system in the proposed. 
+            - For each cooling_system_p in hvac_p: `for cooling_system_p in hvac_p.cooling_system:`
+                - Check if the cooling_system_type != "None": `if cooling_system_p.cooling_system_type != "None":`
+                    - Set virtual_cooling_modeled_check_p to true: `virtual_cooling_modeled_check_p = TRUE`
+            - Check if cooling was modeled in the proposed via the virtual_cooling_modeled_check_p boolean variable and add the hvac system to the list with virtual cooling if it was: `if virtual_cooling_modeled_check_p == TRUE: proposed_hvac_modeled_with_virtual_cooling_list_p = proposed_hvac_modeled_with_virtual_cooling_list_p.append(hvac_p.id)`
+        - Else, the hvac system in the proposed design model is also in the U_RMR: `Else:`
+            - Reset virtual_cooling_modeled_check_p boolean variable: `virtual_cooling_modeled_check_p = FALSE`
+            # Below compares the cooling systems associated with the hvac system across the P_RMR and U_RMR only if cooling was modeled in the proposed. If cooling was modeled in the proposed and NOT in the U_RMR (the cooling system type is None in the U_RMR) then this means virtual cooling was modeled in the proposed. 
+            - For each cooling_system_p in hvac_p: `for cooling_system_p in hvac_p.cooling_system:`
+                - Check if "None" is NOT the cooling_system_type for cooling_system_p, if its not "None" (means cooling was modeled) then get the analogous cooling system in the U_RMR and check if the cooling_system_type is "None" in the U_RMR: `if cooling_system_p.cooling_system_type != "None":`
+                    - Get analogous cooling system in U_RMR: `cooling_system_u = match_data_element(U_RMR,CoolingSystem,cooling_system_p.id)`
+                    - Check if the analogous system in the U_RMR has a cooling system type equal to "None", if it does then set the virtual_cooling_modeled_check_p boolean variable to TRUE : `if cooling_system_u.cooling_system_type == "None": virtual_cooling_modeled_check_p = TRUE`
+            - Check if cooling was modeled in the proposed via virtual_cooling_modeled_check_p boolean variable (and not in the U_RMR) add hvac system to list: `if virtual_cooling_modeled_check_p == TRUE: proposed_hvac_modeled_with_virtual_cooling_list_p = proposed_hvac_modeled_with_virtual_cooling_list_p.append(hvac_p.id)`
+
+**Returns** `return proposed_hvac_modeled_with_virtual_cooling_list_p`  
+
```

Comment:
@claperle we dont seem to have a rule assertion out here.
Do we need to add the following
1. Case 1 - U-RMR has no cooling and P-RMR has been modeled with virtual cooling system which meets the baseline requirmenets = Pass
2. Case 2 - U RMR - all systems ahve H and C and P-RMR = U-RMR (Not Applicable)
3. Case 3- U_RMR has a system with no cooling and P-RMR has also been modeled with no cooling = Fail

Not sure if additional scenarios exisit.

Reply:
@supriyagoel This is a function with no associated rule. I am not following the reason for a rule assertion. The goal of the function is to return a list of proposed hvac systems modeled with virtual cooling.

Reply:
@claperle yes, that makes sense. Should there also be a rule to verify both G3.1 #10 d and G3.1 #10 c.
It could be something simple that calls this function and returns the rule assertion as identified above.
We can wait for @mkarpman to come back before deciding.

Reply:
Discussed on call, no additional RDS needed.